src/app/main/component/shared/components/warning-pop-up/warning-pop-up.component.ts

```diff
@@ -14,9 +16,19 @@ export class WarningPopUpComponent implements OnInit, OnDestroy {
   public popupConfirm: string;
   public popupCancel: string;
   public closeButton = './assets/img/profile/icons/cancel.svg';
+  public deleteButtonVisible: boolean;
   private destroyed$: ReplaySubject<any> = new ReplaySubject<any>(1);
 
-  constructor(private matDialogRef: MatDialogRef<WarningPopUpComponent>, @Inject(MAT_DIALOG_DATA) public data) {}
+  constructor(
+    @Inject(MAT_DIALOG_DATA) public data,
+    public router: Router,
+    public orderService: OrderService,
+    private matDialogRef: MatDialogRef<WarningPopUpComponent>
+  ) {
+    this.orderService.getStepperFinalSubject.subscribe((stepperPosition) => {
```

Comment:
`stepperPosition` it's the incorrect name for that variable
As I understand this subject returns if the stepper is on the final step or not, but not the position of the stepper

Reply:
done.

Reply:
I hope it's finish